src/main/java/org/openmrs/addonindex/service/Index.java

```diff
@@ -27,7 +27,7 @@
 	
 	Collection<AddOnInfoAndVersions> getAllByType(AddOnType type) throws Exception;
 	
-	Collection<AddOnInfoAndVersions> getByTag(String tag) throws Exception;
+	Collection<AddOnInfoSummary> getByTag(String tag) throws Exception;
```

Comment:
Cause I felt it returns more info than needed for our current use case. Let me know what you feel and accordingly I'll change it .